libraries/AP_SmartRTL/AP_SmartRTL.h

```diff
@@ -220,3 +229,9 @@ class AP_SmartRTL {
     // returns true if the two loops overlap (used within add_loop to determine which loops to keep or throw away)
     bool loops_overlap(const prune_loop_t& loop1, const prune_loop_t& loop2) const;
 };
+
+
+namespace AP {
+    AP_SmartRTL *smartrtl();
```

Comment:
I think you're missing the implementation of this function.  If you're trying to use this you may be getting a link failure.

Reply:
1) haven't tried yet....first step is to get the means to get at the SmartRTL function calls from within AP_OSD...and I'm totally just winging it...monkey see monkey do from the link tridge gave me
2) don't know what you think I need to do...help

Reply:
I need to be able to get the number of path points currently used from within AP_OSD_Screen.cpp and the current max number of points (not the param since it could have been modified )...I can do the rest